examples/credential-registry/src/lib.rs

```diff
@@ -521,13 +544,19 @@ impl Deserial for CredentialEvent {
     }
 }
 
+#[derive(Serialize, SchemaType)]
+pub struct InitParams {
+    issuer_metadata: MetadataUrl,
+    storage_address: ContractAddress,
+}
```

Comment:
While we are at it we should also add 
```
#[concordium(size_length = 1)]
schemas: Vec<(CredentialType, SchemaRef)>,
```

to this since you have to add some schemas before issuing any credentials, and this makes setup easier.

Reply:
And of course then call `add_schemas`.